command/mounttune.go

```diff
@@ -0,0 +1,106 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/vault/vault"
+)
+
+// RemountCommand is a Command that remounts a mounted secret backend
+// to a new endpoint.
+type MountTuneCommand struct {
+	Meta
+}
+
+func (c *MountTuneCommand) Run(args []string) int {
+	var defaultLeaseTTL, maxLeaseTTL string
+	flags := c.Meta.FlagSet("mount-tune", FlagSetDefault)
+	flags.StringVar(&defaultLeaseTTL, "default-lease-ttl", "", "")
+	flags.StringVar(&maxLeaseTTL, "max-lease-ttl", "", "")
```

Comment:
Can we have a `-set` flag that switches from reading the tune values (default) to setting the tune values (with -set)? Is there some other way to query the tune values?